lib/definitions/time.js

```diff
@@ -0,0 +1,71 @@
+var time
+
+time = {
+  ms: {
+    name: {
+      singular: 'Millisecond'
+    , plural: 'Milliseconds'
+    }
+  , to_anchor: 1/1000
+  }
+, s: {
+    name: {
+      singular: 'Second'
+    , plural: 'Seconds'
+    }
+  , to_anchor: 1
+  }
+, min: {
+    name: {
+      singular: 'Minute'
+    , plural: 'Minutes'
+    }
+  , to_anchor: 60
+  }
+, h: {
+    name: {
+      singular: 'Hour'
+    , plural: 'Hours'
+    }
+  , to_anchor: 3600
```

Comment:
I think these would be better expressed in terms of their components, like `60 * 60`. Otherwise in the following units, it's really hard to tell what you were thinking when you say "one month is 2419200 seconds".